### PR TITLE
Rc 0.10.0

### PR DESCRIPTION
**Pull request by menghaozhang**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -8,16 +22,19 @@
...
-- Configure to show currency symbol on overview with `shouldShowCurrencySymbolInStatisticsTotal` in `FinanceOverviewViewController`
-- Configure `UISegmentedControl` appearance with appearance proxy.
-- Updated date formats on transactions.
+    - Configure the check box radius with `checkBoxIconBackgroundCornerRadiusFactor` in `ColorProvider`.
+    - Configure inverted button with `secondaryButtonTitle` in `ColorProvider`.
+    - Configure pill button with `tertiaryButtonTitle` and `tertiaryButtonBackground` in `ColorProvider`.
+    - Configure monthly picker chevron button with `chevronButton` and `chevronButtonDisabled` in `ColorProvider`.
+    - Configure to show currency symbol on overview with `shouldShowCurrencySymbolInStatisticsTotal` in `FinanceOverviewViewController`
+    - Configure `UISegmentedControl` appearance with appearance proxy.
+- Updated date formats on transactions - Always Italian locale for date formatting on Enel-X app.
```
Comment (lilSwifty) [inline CHANGELOG.md]: Should this be here?

File: CHANGELOG.md [hunk 2]
```diff
@@ -8,16 +22,19 @@
...
 - Removed the amount rounding for every currency except SEK and NOK.
 - Fixed the issue where latest transactions in overview/detail view have different dates.
 
+- Updated the Enel-X example app with the usage of above changes and with `UISegmentedControl`, `UINavigationbar` appearance configuration.
+- For more information, please see the usage in Enel-X example app.
```
Comment (lilSwifty) [inline CHANGELOG.md]: Same here.

**Review by lilSwifty — CHANGES_REQUESTED** [review]
Why are we doing changes in the earlier versions? And also mentioning Enel-X in public repo?

File: CHANGELOG.md (hunk 1 above)
Comment (menghaozhang) [reply CHANGELOG.md]: I think you are checking some outdated commit

**Comment by menghaozhang** [comment]
> Why are we doing changes in the earlier versions? And also mentioning Enel-X in public repo?

Think you are not checking the latest, it was copied and I removed it manually before making the PR

**Review by lilSwifty — APPROVED** [review]

**Review by kasper-lahti — APPROVED** [review]

**Pull request merged**